consume-more-stuff/server/server.js

```diff
@@ -47,14 +49,53 @@ app.use(bodyParser.urlencoded({ extended: true }));
 //     });
 //   })
 
-app.use("/users", usersRouter);
-app.use("/items", itemsRouter);
-app.use("/categories", categoriesRouter);
+app.use("/api/users", usersRouter);
+app.use("/api/items", itemsRouter);
+app.use("/api/categories", categoriesRouter);
+app.use((req,res,next)=>{
```

Comment:
Mentioned this to Jen already so coordinate whose PR will fix but this is a security vulnerability of allowing all domains to submit requests to this domain.